app/Policies/ResourcePolicy.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace App\Policies;
+
+use App\User;
+use App\Resource;
+use Illuminate\Auth\Access\HandlesAuthorization;
+
+class ResourcePolicy
+{
+    use HandlesAuthorization;
+
+    /**
+     * Determine whether the user can view the resource.
+     *
+     * @param \App\User $user
+     * @param \App\Response $response
+     * @return mixed
+     */
+    public function view(User $user, Resource $resource)
+    {
+        return $user->isAdmin();
```

Comment:
Any user can view a resource.
```suggestion
        return true;
```